crates/sel4/src/arch/x86/arch/x64/invocations.rs

```diff
@@ -91,8 +91,57 @@ impl<C: InvocationContext> PageTable<C> {
     }
 }
 
-// TODO
-impl<C: InvocationContext> IrqControl<C> {}
+impl<C: InvocationContext> IrqControl<C> {
```

Comment:
These should be for IA32 as well right? Not sure if IA32 is supported by rust-seL4 yet. If it is then I can move these to some generic x86 `invocations.rs` file.

Reply:
Yes, the C initialiser seems to guard these with `CONFIG_ARCH_X86`.

Reply:
There is actually no plan currently for this project to support IA32. The reason for this is that we are unaware of any demand for it, and supporting it would add significant complexity to the `sel4-sys` crate.

Here is the guard against IA32:

https://github.com/seL4/rust-sel4/blob/d3341eacd9da15c2edf96ac2c05afbb67506152b/crates/sel4/sys/build/main.rs#L79-L81

Reply:
Ah okay sure.